modules/modes/ModeBrowse.js

```diff
@@ -64,5 +66,37 @@ export class ModeBrowse extends AbstractMode {
     console.error('error: do not call modeBrowse.sidebar anymore');   // eslint-disable-line no-console
   }
 
+  /**
+   * _hover
+   * Hover changed cursor styling based one what geometry is hovered
+   */
+  _hover(eventData) {
+    // Get the current context and graph
+    // const cursorPoint = this.textures.get('vertex');
+    // const cursorSpritePoint = new PIXI.Sprite(cursorPoint);
+
+    const context = this.context;
+    const graph = context.graph();
+    // Get the target and associated datum
+    const target = eventData.target;
+    const datum = target && target.data;
+    // Check if the datum is an entity in the graph
+    const entity = datum && graph.hasEntity(datum.id);
+    // Get the geometry of the entity, if it exists
+    const geom = entity && entity.geometry(graph);
+    // Change the cursor of the document body based on the geometry type
+    if (geom && geom === 'vertex') {
```

Comment:
In general, when decision making in an if/else, try to limit things to a single assignment statement at the end (if possible). 

```
let cursorPath;
if (vertex) {
cursorPath = 'url(/img/cursor/cursor-select-vertex.png), auto';
} 
/* ... other clauses cut for brevity  ... */ 
else {
    cursorPath = 'url(/img/cursor/cursor-grab.png),auto';
}

document.body.style.cursor = cursorPath
}

```